src/payment/strategies/googlepay/googlepay-payment-processor.spec.ts

```diff
@@ -232,10 +232,15 @@ describe('GooglePayPaymentProcessor', () => {
             const googlePaymentDataMock = getGooglePaymentDataMock();
             googlePaymentDataMock.paymentMethodData.info.billingAddress = getGooglePayAddressMock();
 
-            await processor.initialize('googlepay');
-            await processor.handleSuccess(googlePaymentDataMock);
+            try {
+                await processor.initialize('googlepay');
+                await processor.handleSuccess(googlePaymentDataMock);
 
-            expect(googlePayInitializer.parseResponse).toHaveBeenCalled();
+                expect(googlePayInitializer.parseResponse).toHaveBeenCalled();
+            } catch (error) {
+                expect(error).toBeInstanceOf(MissingDataError);
```

Comment:
using try/catch in a unit tests is discouraged because there's no guarantee the exception is actually being thrown.

instead, you should use
```js
expect(() => functionToTest()).toThrow(new MissingDataError(MissingDataErrorType.MissingBillingAddress))
```

Reply:
hi @capsula4 I did the changes. the unit tests where billing address does not exist, already exists in the file, so i removed the try catch, from the googlepay-payment-processor.spec.ts, i just modify the googlepay.mock.ts adding lastname to the "name" because its needed.